docs/plugins/jenkins.md

```diff
@@ -4,15 +4,36 @@ This plugin installs [jenkins](https://jenkins.io) in an existing Kubernetes clu
 
 ## Usage
 
-NOTICE:
+### NOTICE
 
-Create the data directory manually:
+#### Production Environments
+Please be sure to change the `storageClass` in the configuration item to an existing storage class.
+
+#### Test Environments
+If you want to **test** the plugin in a **local environment**:
+
+1. Please change the `test_env` to `true` in the config file.
+2. Create the data directory manually in the host where kubernetes is running:
 
 ```bash
+# If you run k8s and dtm in the same host
 mkdir -p ~/data/jenkins-volumes/
 chown -R 1000:1000 ~/data/jenkins-volumes/
+
+-------------------------
+
+# If you run k8s and dtm in different hosts, such as run k8s in docker
```

Comment:
It's better to tell users how to execute `chown` with Minikube. e.g. `minikube ssh & mkdir ... `

Reply:
Thank you very much! ❤️